src/routing.ts

```diff
@@ -30,6 +33,12 @@ export const initRouting = ({
     routing,
     hasCors: !!config.cors,
     onEndpoint: (endpoint, path, method, siblingMethods) => {
+      // @todo this is too implicit
+      const hasUpload = endpoint.getMimeTypes("input").includes(mimeMultipart);
+      if (hasUpload && uploader) {
+        // @todo that requires error handling
+        app[method](path, uploader);
```

Comment:
this should actually go even further, after the middlewares to enable authentication by the proprietary middlewares